ui/nuxeo-layout.js

```diff
@@ -168,6 +178,15 @@ import './nuxeo-error.js';
         });
       }
     }
+
+    _applyAutoFocus(element) {
+      if (element && element.root && !element.root.activeElement) {
+        const focusableElement = Array.from(element.root.children).find((el) => el.autofocus);
+        if (focusableElement) {
+          focusableElement.focus();
```

Comment:
Perhaps this should be a recursive method, just like `_getValidatableElements `, since you can have multiple shadow roots between this element's root and the closest auto-focusable input.

Reply:
Fair enough. I'll perform that change ;)